core/global_enum.py

```diff
@@ -32,10 +32,7 @@ class ConfigurationNameEnum(Enum):
     SUBJECTS_SEPARATOR_ROLE = "subjectsSeparator"
     GROUP_CATEGORY = "groupCategory"
     SUBJECTS_CATEGORY = "subjectsCategory"
-    STUDY_CATEGORY = "studyCategory"
-    GAMING_CATEGORY = "gamingCategory"
     STUDY_JOIN_VOICE_CHANNEL = "studyVoiceChannel"
```

Comment:
Not needed anymore?

Reply:
No we have [TEMP_CHANNELS ](https://github.com/marcelropos/HM-DiscordBot/blob/f4fedba563c01aec93ff617ed5a95e12738359a1/core/global_enum.py#L18) now.

Reply:
Yes so `STUDY_JOIN_VOICE_CHANNEL` can go right?

Reply:
[STUDY_JOIN_VOICE_CHANNEL ](https://github.com/marcelropos/HM-DiscordBot/blob/2e535d74308935a07acb0b48573f4d8339c4b949/core/global_enum.py#L37) & [GAMING_JOIN_VOICE_CHANNEL ](https://github.com/marcelropos/HM-DiscordBot/blob/2e535d74308935a07acb0b48573f4d8339c4b949/core/global_enum.py#L38)
Is deprecated and replaced by [TEMP_CHANNELS_CONFIGURATION ](https://github.com/marcelropos/HM-DiscordBot/blob/2e535d74308935a07acb0b48573f4d8339c4b949/core/global_enum.py#L20).

Reply:
Yeah so delete [STUDY_JOIN_VOICE_CHANNEL](https://github.com/marcelropos/HM-DiscordBot/blob/2e535d74308935a07acb0b48573f4d8339c4b949/core/global_enum.py#L37)

Reply:
Done, please wait for push.